From c6c19fb9aef732b9ba1d1de8e14396188a0a25f1 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Thu, 14 Mar 2013 17:58:01 +0800
Subject: [PATCH 162/223] ENGR00252064-3 camera: ov5640: fix missed setting
 for light_frequency

Confirmed from OV, the missed setting for light_frequency need to be added.

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 drivers/media/video/mxc/capture/ov5640.c      |    1 +
 drivers/media/video/mxc/capture/ov5640_mipi.c |    1 +
 2 files changed, 2 insertions(+)

diff --git a/drivers/media/video/mxc/capture/ov5640.c b/drivers/media/video/mxc/capture/ov5640.c
index 3195377..fabbc23 100644
--- a/drivers/media/video/mxc/capture/ov5640.c
+++ b/drivers/media/video/mxc/capture/ov5640.c
@@ -828,6 +828,7 @@ static int ov5640_get_light_freq(void)
 			light_frequency = 50;
 		} else {
 			/* 60Hz */
+			light_frequency = 60;
 		}
 	}
 
diff --git a/drivers/media/video/mxc/capture/ov5640_mipi.c b/drivers/media/video/mxc/capture/ov5640_mipi.c
index 04f5ee7..a373e5b 100644
--- a/drivers/media/video/mxc/capture/ov5640_mipi.c
+++ b/drivers/media/video/mxc/capture/ov5640_mipi.c
@@ -904,6 +904,7 @@ int OV5640_get_light_freq(void)
 			light_freq = 50;
 		} else {
 			/* 60Hz */
+			light_freq = 60;
 		}
 	}
 	return light_freq;
-- 
1.7.10.4

